scavenger-agent-java/src/integrationTest/java/integrationTest/support/AgentRunner.java

```diff
@@ -0,0 +1,85 @@
+package integrationTest.support;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.Callable;
+import java.util.stream.Collectors;
+
+public class AgentRunner implements Callable<String> {
+    private final String javaPath;
+    private final String classpath;
+    private final String agentPath;
+
+    private String configFilePath;
+    private String cmdLineArgs;
+
+    public AgentRunner(String javaPath, String classpath, String agentPath) {
+        this.javaPath = javaPath;
+        this.classpath = classpath;
+        this.agentPath = agentPath;
+    }
+
+    public void setConfigFilePath(String configFilePath) {
+        this.configFilePath = configFilePath;
+    }
+
+    public void setConfig(Properties properties) {
+        cmdLineArgs = properties.entrySet().stream()
+            .map(entry -> entry.getKey() + "=" + entry.getValue())
+            .collect(Collectors.joining(";"));
+    }
+
+    @Override
+    public String call() throws IOException, InterruptedException, RuntimeException {
+        List<String> command = buildCommand();
+        Process process = new ProcessBuilder().command(command).redirectErrorStream(true).start();
+        String output = collectProcessOutput(process.getInputStream());
+        int exitCode = process.waitFor();
+        if (exitCode != 0) {
+            throw new RuntimeException(String.format("Could not execute '%s': %s%nExit code=%d", command, output, exitCode));
+        }
+        return output;
+    }
+
+    private List<String> buildCommand() {
+        String cp = classpath.endsWith(":") ? classpath.substring(0, classpath.length() - 2) : classpath;
+
+        List<String> command = new ArrayList<>();
+        command.add(javaPath);
+        if (cmdLineArgs != null) {
+            command.add("-javaagent:" + agentPath + "=" + cmdLineArgs);
+        } else {
+            command.add("-javaagent:" + agentPath);
+        }
+        command.add("-cp");
+        command.add(cp);
+        command.add("-Djava.util.logging.config.file=src/integrationTest/resources/logging.properties");
+        command.add("-Duser.language=en");
+        command.add("-Duser.country=US");
```

Comment:
This configuration seems to be unused.

Reply:
This keeps log format consistent. Notice log level on two screenshot differs.

With options:
<img width="630" alt="스크린샷 2023-06-05 11 45 46" src="https://github.com/naver/scavenger/assets/4951985/6f96f4d9-0d9b-4129-b140-cd7fef7986cb">

Without options:
<img width="550" alt="스크린샷 2023-06-05 11 46 19" src="https://github.com/naver/scavenger/assets/4951985/f33c5f71-fa04-44df-94be-e482db863ce7">